blocks/ticket/ticket.ts

```diff
@@ -150,10 +150,11 @@ export class Ticket {
                       SHORT_TXT: ["TXT"],
                       LONG_TXT: ["TXT"],
                       NUM: ["NUM", "SLDR"],
-                      SEL: ["SEL", "RAD", "CBX", "ACT"],
```

Comment:
Why do we need to remove this?

Reply:
i think that was done by mistake fixed now